src/NuGet.Services.BasicSearch/ServiceEndpoints.cs

```diff
@@ -15,6 +15,7 @@ public class ServiceEndpoints
             var skip = GetSkip(context);
             var take = GetTake(context);
             var includePrerelease = GetIncludePrerelease(context);
+            var includeSemver2 = GetIncludeSemver2(context);
```

Comment:
The issue is not the meaning of the parameter, but the fact that it is bool. IMO the code at this level should be more general, and keep the SemVer value as a NuGetVersion.